src/Operation/Aggregate.php

```diff
@@ -303,13 +308,25 @@ private function createCommand(Server $server, $isCursorSupported)
             $cmd['hint'] = is_array($this->options['hint']) ? (object) $this->options['hint'] : $this->options['hint'];
         }
 
+        if (isset($this->options['maxAwaitTimeMS'])) {
+            $cmdOptions['maxAwaitTimeMS'] = $this->options['maxAwaitTimeMS'];
+        }
+
+        if (isset($this->options['readConcern'])) {
+            $cmd['readConcern'] = \MongoDB\read_concern_as_document($this->options['readConcern']);
+        }
+
+        if (isset($this->options['writeConcern'])) {
+            $cmd['writeConcern'] = \MongoDB\write_concern_as_document($this->options['writeConcern']);
+        }
```

Comment:
Assuming this commit was rebased, `readConcern` and `writeConcern` were moved in https://github.com/mongodb/mongo-php-library/commit/091d8bd9a7ad904b2030206089a6521ef7de925c#diff-3b2e2c1a4b1425910f13159568854106L302 to a separate `createOptions()` method. If this commit is adding them back, it's likely that these got put back when selecting the `maxAwaitTimeMS` addition (git likely presented them all as one addition). This would have required splitting the diff during the rebase, which I'm happy to walk through and demonstrate.